tools/pkg/repo/create.py

```diff
@@ -530,7 +532,14 @@ def _create_repo_file(create_repo_path, url_suffix):
 
     _create_repo_file(repo_file_path, f"minor/{salt_version}")
 
-    if not nightly_build_from:
+    if nightly_build_from:
+        nightly_latest_repo_file_path = create_repo_path.parent / "nightly_latest.repo"
+        _create_repo_file(nightly_latest_repo_file_path, "minor/nightly", "latest")
```

Comment:
```suggestion
        nightly_latest_repo_file_path = create_repo_path.parent / "nightly_latest.repo"
        _create_repo_file(nightly_latest_repo_file_path, "minor/nightly", "latest")
```
About this one, whats wrong with `https://repo.saltproject.io/salt-dev/3006.x/latest/salt/py3/redhat/9/x86_64/minor/nightly.repo` ? why do we have to have a `nightly_latest.repo`?

Reply:
I wasnt sure if you would want to change the path in the existing nightly repofile

Reply:
current file contents:
```
[salt-nightly-repo]
name=Salt repo for RHEL/CentOS 9 PY3
baseurl=https://repo.saltproject.io/salt-dev/3006.x/2023-08-04/salt/py3/redhat/9/x86_64/minor/3006.1+298.g08482ed6f3
skip_if_unavailable=True
priority=10
enabled=1
enabled_metadata=1
gpgcheck=1
gpgkey=https://repo.saltproject.io/salt-dev/3006.x/2023-08-04/salt/py3/redhat/9/x86_64/minor/3006.1+298.g08482ed6f3/SALT-PROJECT-GPG-PUBKEY-2023.pub
```

the proprosed contents of the nightly_latest.repo file urls would change from:
`https://repo.saltproject.io/salt-dev/3006.x/2023-08-04/salt/py3/redhat/9/x86_64/minor/3006.1+298.g08482ed6f3`
to
`https://repo.saltproject.io/salt-dev/3006.x/latest/salt/py3/redhat/9/x86_64/minor/nightly`

the reason for an additional file.
one (current file) would point to the specific datefile of the build, which would become invalid once that date folder is dropped.
one (new file) would always point to latest path and be a valid repo url.

so users who install nightlys with the nightly_latest.repo would get updates daily with `yum update`. users using nightly.repo will eventually be pointing to a non-existant repo location

Reply:
paths adjusted to `minor/latest`